agent/job_runner.go

```diff
@@ -440,6 +441,7 @@ func (r *JobRunner) createEnvironment() ([]string, error) {
 	// Add options from the agent configuration
 	env["BUILDKITE_CONFIG_PATH"] = r.conf.AgentConfiguration.ConfigPath
 	env["BUILDKITE_BUILD_PATH"] = r.conf.AgentConfiguration.BuildPath
+	env["BUILDKITE_REPOS_PATH"] = r.conf.AgentConfiguration.ReposPath
```

Comment:
nit: Maybe it would be a bit more unambiguous here to call it something like `BUILDKITE_REPO_CACHE_PATH` or the like?

Otherwise, I could see how one might not immediately see the functional difference between this var for setting where mirrored copies live and `BUILDKITE_BUILD_PATH` for setting where the working copies live.

Reply:
I agree (but `BUILDKITE_REPOS_CACHE_PATH` - pluralise).

Reply:
I wonder if it is actually a "cache" though. I'm not entirely sure that word adds clarity that justifies the additional length. Will think it over.